packages/components/src/toggle-group-control/toggle-group-control-option-base/styles.ts

```diff
@@ -12,6 +12,12 @@ import type {
 	ToggleGroupControlProps,
 	ToggleGroupControlOptionBaseProps,
 } from '../types';
+import { BACKDROP_BG_COLOR } from '../toggle-group-control/styles';
```

Comment:
The changes in this style file is to allow a pressed button's black background to remain "static", unlike the animated roving background in the single-select variant.